DQM/HLTEvF/python/HLTObjectsMonitor_B2G_cfi.py

```diff
@@ -0,0 +1,83 @@
+import FWCore.ParameterSet.Config as cms
+
+b2gObjects = cms.VPSet(
+       cms.PSet(
```

Comment:
you are suggesting to make explicit only the parameters which need to be overwritten w.r.t. a "template" / "standard" / "default" PSet ?
I tend to agree, but if it is not too bad, I would prefer to have all parameters explicitly set for each trigger path